scripts/shim.js

```diff
@@ -35,6 +35,10 @@ fs.writeFileSync(
     `@import url("../../styles/ux-aspects.min.css");`
 );
 
+fs.writeFileSync(
+    path.join(cssPath, 'hpe-icons.min.css'),
+    `@import url("../../styles/hpe-icons.min.css");`
+);
```

Comment:
This should not be added here; this script is for backward compatibility only (and no longer used to 2.0).